### ISSUE TITLE
Advanced video saving

### ISSUE THREAD

**Issue by OffifialMITX** [vijai1996/screenrecorder#18]
It would be a nice feature if you could cut the videos (just a simple "cut away the beginning and the end" in sec.) or at least had a countdown like 3 sec. after starting the recording and thelast 5 sec. not being saved anymore by standard.

**Comment by vijai1996** [comment]
Thanks for your continuous support and suggestions on the app. May I know whats the reason for cutting the video?

**Comment by OffifialMITX** [comment]
Of course, imagine, for example, you are going to make a screencast of how an app works and if you did everything perfect, the way screenrecorder works yet, in the beginning there are like 5-7 seconds until you find the app in your launcher, or at least 3 seconds if you're well prepared.

It's not much time itself but enough for the viewer's attention to be dropped and this part is completely worthless, just like the part when you're done and are about to stop the recording.

So, even if everything went perfect, you (or, at least me) still have to cut the video using a computer.

This is also why the direct share function is completely useless for me so it would be great if you'd give the option to disable it.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
I will look into adding a time lag before starting to record (probably with a setting to control time) in a future release but cutting the video in the end is something I wouldnt agree to. The time may vary and its not possible to predetermine what amount of time is to be cut from the end.

**Comment by OffifialMITX** [comment]
Oh, you got ot wrong, I meant either a countdown or an option to cot the videos manually.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
If I understood correct, you say I either add a setting to delay recording start or a in-app video editor to cut the videos?

**Comment by OffifialMITX** [comment]
That's exactly what I meant. Maybe it would be worth it to implement both.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
I will be adding video editor via some library soon in the upcoming update. A delay for video recording start will be kept on hold for now

**Comment by vijai1996** [comment]
The feature has been added and should reflect in the next F-Droid update. Closing the issue now
